NineChronicles.Headless/GraphTypes/ActionQuery.cs

```diff
@@ -401,6 +402,30 @@ public ActionQuery(StandaloneContext standaloneContext)
                     return Encode(context, action);
                 }
             );
+            Field<NonNullGraphType<ByteStringType>>(
+                "activateAccount",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<StringGraphType>>
+                    {
+                        Name = "activationCode",
+                        Description = "Activation code that you've get."
+                    }
+                    ),
+                resolve: context =>
+                {
+                    var activationCode = context.GetArgument<string>("activationCode");
+                    var activationKey = ActivationKey.Decode(activationCode);
+                    if (standaloneContext.BlockChain!.GetState(activationKey.PendingAddress) is Dictionary dictionary)
+                    {
+                        var pending = new PendingActivationState(dictionary);
+                        var action = activationKey.CreateActivateAccount(pending.Nonce);
+                        pending.Verify(action);
```

Comment:
This line seems not to affect anything. If it was to check whether the activation code is valid or not, it should throw an execution error like the below code (I'm not sure the below code is able to be run because I didn't test it.)
```suggestion
                        if (pending.Verify(action))
                        {
                            throw new ExecutionError("Invalid activation code. Failed to verify pending activation state.")
                        }
```